lib/mv_opentelemetry/ecto.ex

```diff
@@ -75,18 +66,32 @@ defmodule MvOpentelemetry.Ecto do
     ]
 
     all_attributes = result ++ base_attributes ++ time_attributes(measurements)
-
     span_name = name(config, event, source)
-    span_opts = %{start_time: start_time, attributes: all_attributes}
+    span_opts = %{start_time: start_time, attributes: all_attributes, kind: :client}
+
+    span = Tracer.start_span(span_name, span_opts)
+
+    case query_result do
+      {:error, error} ->
+        OpenTelemetry.Span.set_status(span, OpenTelemetry.status(:error, format_error(error)))
 
-    Tracer.start_span(span_name, span_opts)
-    |> Span.end_span()
+      {:ok, _} ->
+        :ok
+    end
+
+    Span.end_span(span)
 
     :ok
   end
 
+  defp format_error(%{__exception__: true} = exception) do
+    Exception.message(exception)
+  end
+
+  defp format_error(_), do: ""
```

Comment:
This one i'm not entirely sure - can't error be something rather than exception? 
- If it can then why its ignored and not put into status
- If it cannot then why its ignored and not reported anywhere as an actual error

Reply:
Hmm, maybe `inspect(error)`? It will always give a string and it has more info than empty string anyway

Reply:
That is not a good idea for performance reasons - We cannot know if it is a 4-levels nested 32 field map, a very long list or something entirely different.

Reply:
Ok!